server/routes/checkAnswers.js

```diff
@@ -47,3 +150,64 @@ module.exports = function Index({ formService, authenticationMiddleware }) {
 
   return router
 }
+
+// utility function - convert to string where input MAY be an array
+
+const convertNestedArraysToString = allElements => {
+  if (Array.isArray(allElements)) {
+    let finalString = ''
+    allElements.forEach(element => {
+      finalString += `${element}, `
+    })
+    finalString = finalString.slice(0, -2)
+    return finalString
+  }
+  return allElements
+}
+
+// utility function - convert to string when input WILL be nested objects
+const convertArrayOfObjectsToString = (dataArray = []) => {
+  let finalString = ''
+  dataArray.forEach(element => {
+    finalString += `${element.name}, `
+  })
+  return finalString.slice(0, -2)
+}
+
+// titlecase all person names
+const toTitleCase = (str = '') => {
```

Comment:
we have a method in utils that we could use instead of this: `properCaseName`

Reply:
agreed and have replaced occurrences of toTitleCase with properCaseName

Reply:
fair enough, that has actually stung me a couple of times.
I think this could be moved in to utils and tested independently - but could do this in a follow up?